src/Frontend/Page/Delegation.hs

```diff
@@ -181,22 +182,18 @@ viewDelegationNetwork (fromMaybe (DScopeIdeaSpace SchoolSpace) -> scope) = do
 
 delegationInfos :: DScope -> EQuery DelegationNetwork
 delegationInfos scope = do
-    delegations <- findImplicitDelegationsByScope scope
+    delegations <- Map.toList <$> findImplicitDelegationsByScope scope
 
     -- Create delegations
-    let mkGraphNode (de, _, dees) = (unDelegate de, unDelegate de, unDelegatee <$> dees)
+    let mkGraphNode (de, dees) = (unDelegate de, unDelegate de, unDelegatee . snd <$> dees)
 
     -- Build graphs and graph handler functions
     let graphNodes = fixLeaves $ mkGraphNode <$> delegations
```

Comment:
ok, got it.  if we didn't have `delegations` in the form preferred by you, this is where would have to group it at the lastest.

so why don't we do it here?  make it an independent function, obviously, but call that function here?  then we wouldn't have to flatten it back below.